debian/control.top.in

```diff
@@ -10,11 +10,11 @@ Build-Depends:
     @KERNEL_HEADERS@,
     @MODUTILS_DEPENDS@,
     @EXTRA_BUILD@,
-    docbook-xsl,
-    asciidoc,
```

Comment:
I believe every build uses asciidoc at the moment, to generate man pages, as these are installed no matter if documentation is installed or not.

Reply:
Fixed in https://github.com/LinuxCNC/linuxcnc/pull/2647 .

Reply:
I consider this patch as a transition to your superior set of patches.

Reply:
If so, perhaps that change should be taken out of this pull request?

Reply:
hm, look like the sequencing of the last two comments are confusing.  I tried to suggest to remove the changes also done in #2647 from this pull request, to avoid conflicts between the two.

Reply:
Done. Thank you.